api/Request.json

```diff
@@ -1436,10 +1436,10 @@
               "version_added": "41"
             },
             "safari": {
-              "version_added": "11.1"
```

Comment:
This "11.1" came from https://github.com/mdn/browser-compat-data/pull/2858/files#diff-51792689c76b160a652e7eb8956b5e4f where a lot of other `referrerPolicy` attributes updated. However, as far as I can tell from `git log -S referrerPolicy -- '*.idl'` in WebKit, those were implemented separately, they're not really connected.